### ISSUE TITLE
Show note to recipient on shares more prominent

### ISSUE THREAD

**Issue by bpcurse** [nextcloud/server#13874]
**Is your feature request related to a problem? Please describe.**
If a note on shared content is important, the actual way to display it as a small bubble on the top right is not sufficient. Most people opening the share link probably won't even notice it is there.

**Describe the solution you'd like**
The note could be displayed in a way similar to the first run wizard. Ideally the user who shares the link can choose between the actual way (small bubble) and the forced display e.g. with a checkbox "force display".

**Comment by MorrisJobke** [comment]
cc @nextcloud/designers

**Comment by ariko861** [comment]
A javascript popup window could appear once the page has loaded. This could be set as an option in the sharing options.

**Comment by skjnldsv** [comment]
@jancborchardt why not use the same design as the readme header thing? :)

**Comment by jancborchardt** [comment]
> @jancborchardt why not use the same design as the readme header thing? :)

Yup, good point, that’s much more obvious and reusable. :) Let’s do that. :+1:

**Comment by juliushaertl** [comment]
The current note button/popover is registered [here](https://github.com/juliushaertl/server/blob/719f9288b97bcd6c9de42d0eb682e11a38ab864b/apps/files_sharing/js/public_note.js#L24-L30). It needs to be replaced with a plugin to the file list that registers a header section as done in the text app: https://github.com/nextcloud/text/blob/master/src/helpers/files.js#L158-L199

**Comment by alexbijma** [comment]
may'be the rich workspace could be used for this, viewing it readonly?

**Comment by juliushaertl** [comment]
Rich workspace won't work here since the file would be the same readme.md for all shares while the note can be different on each share link.

**Comment by Practicalbutterfly5** [comment]
Hello, is this feature still in development?

**Comment by moritzthecat** [comment]
@willemb2 just have similar issue - even could not find the small bubble at all. looks this gets disabled in case hide download is checked. #46189 



#46189
